network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -20,9 +20,10 @@
           {% if USE_CLOUDINARY %}
             src="{% cloudinary_url product.cloudinary_image quality=70 fetch_format="auto" crop="fit" width=736 %}"
           {% else %}
-            src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image }}"
+            {% image product.image width-250 as img %}
+            src="{{ img.url }}"
```

Comment:
This image is a ForeignKey to a Wagtail Image instead of a FIleField (the migration from old PNI products to ForeignKeys will come in a future ticket).